src/fetcher/index.js

```diff
@@ -0,0 +1,17 @@
+import Api from '@/js/RequestApi';
+
+const api = new Api();
+const key = process.env.APP_API_KEY;
+const apiKey = '&api_key=' + key;
+const apiString = 'http://ws.audioscrobbler.com/2.0/?method=';
+const format = '&format=json';
+
+export const lastFm = ({
+  allTrack: apiString + 'track.getTags&api_key=' + apiKey + '&artist=AC/DC&track=Hells+Bells&user=RJ' + format,
+  similarTrack: apiString + 'track.getsimilar&artist=cher&track=believe&api_key=' + apiKey + format,
```

Comment:
it's just object with duplication of apiKey, apiString and other parameters. It's a mistake